ndk-sys/src/lib.rs

```diff
@@ -51,3 +51,7 @@ extern "C" {}
 #[cfg(all(feature = "audio", target_os = "android"))]
 #[link(name = "aaudio")]
 extern "C" {}
+
+#[cfg(all(feature = "midi", target_os = "android"))]
+#[link(name = "amidi")]
```

Comment:
Noticed that I named this `libmidi.so` in #399 the module-level doc-comment of `media_error.rs`. Can you fix that for me (should be `libamidi.so` apparently) as part of this PR? :pray: